js/snake.js

```diff
@@ -140,3 +146,22 @@ function loop(game, ctx) {
     game.update();
     game.render(ctx);
 }
+
+class Score {
+    constructor() {

```

Comment:
I would probably pass the html element `id` as constructor argument here

Reply:
Score class removed